scoped/post/create-post.md

```diff
@@ -0,0 +1,46 @@
+**Name**
+Create post
+
+----
+
+**Actor** 
+Signed-in user
+
+----
+
+**Subject area**
+Any
+
+----
+
+**Preconditions** 
+User is signed in.
+
+----
+
+**Termination outcome**
+User is on question/post page for the now-posted item.
+
+----
+
+**Basic flow**
+
+- User clicks on "ask question" / "post" / etc button (name can vary by category).
+
+- New page has:
+  - for first-time poster, intro message w/link to help (just-in-time help; can be simple or omitted for MVP)
+  - "New post in (category name)" - header to reinforce where you are
+  - places to enter title, body, tags
+
+- User fills out form.  (For MVP we aren't doing any dupe-detection, auto-review, special guidance.)
```

Comment:
What is this form? we should probably link to a feature here? We need to breakdown what functionality of what the form will (and will not) contain for MVP.

Im assuming you mean the WYSIWYG editor here

Reply:
what should a user expect to be able to do for MVP in this form? This could be a simple bulleted list - it doesnt have to be prose.

Reply:
Sorry, by "form" I just meant the "new post" page with textboxes for title, body, and tags.  (What should I call it instead of "form"?)